src/main/java/com/propertydekho/infoservice/models/PropFilterableSortableData.java

```diff
@@ -0,0 +1,41 @@
+package com.propertydekho.infoservice.models;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+public class PropFilterableSortableData {
+    @JsonProperty("prop_id")
+    private String propID;
+    @JsonProperty("prop_name")
+    private String propName;
+    @JsonProperty("prop_price")
+    private double propPrice;
+    @JsonProperty("budget_max")
+    private double max_budget;
```

Comment:
@Divsjains6021 Java Variable should be in camel case. Plus see the json variable name(make it max_budget)

Reply:
changed in next commit..